app/src/main/java/io/github/droidkaigi/confsched2017/util/ContextUtil.java

```diff
@@ -0,0 +1,20 @@
+package io.github.droidkaigi.confsched2017.util;
+
+import android.content.Context;
+import android.content.ContextWrapper;
+
+/**
+ * @author KeithYokoma
+ */
+public final class ContextUtil {
+
+    private ContextUtil() {
+        throw new AssertionError("no instance");
+    }
+
+    public static Context getBaseContext(Context context) {
```

Comment:
🙋 
Is this util class necessary? 🤔 I think this util method is used in few places.